store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9526,5 +9526,10 @@ TODO: delete them permanently from here
   <desc>port number on which the remote IMAP SSL server should listen</desc>
 </attr>
 
+<attr id="3017" name="zimbraReverseProxyUpstreamFairShmSize" type="integer" min="32" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="8.8.1" requiresRestart="nginxproxy">
```

Comment:
I think instead of using type="integer", you should use type="long" because can also contain human readable memory unit values: ex 1KB => 1024 bytes